src/main.rs

```diff
@@ -15,6 +15,8 @@ use std::fs::{self};
 mod constants;
 mod token;
 
+use galoy_cli::types::*;
```

Comment:
`main.rs` is completely bloated. It should just call into something that gets exposed via lib - which is where the use cases should be defined.

IMO this project is getting large enough that a purely design - improvement / refactoring iteration should be executed to improve things before implementing further complex use cases.

At this stage it should still be fairly simple to clean things up - later that might not be the case anymore.